snap-core/src/main/java/org/esa/snap/core/dataio/geocoding/util/InterpolatorFactory.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2021.  Brockmann Consult GmbH (info@brockmann-consult.de)
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the Free
+ * Software Foundation; either version 3 of the License, or (at your option)
+ * any later version.
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, see http://www.gnu.org/licenses/
+ */
+
+package org.esa.snap.core.dataio.geocoding.util;
+
+import org.esa.snap.runtime.Config;
+
+import java.util.prefs.Preferences;
+
+public class InterpolatorFactory {
+
+    final static String SYSPROP_INTERPOLATOR_GEODETIC = "snap.core.geocoding.interpolator.geodetic";
```

Comment:
Better use a property like "snap.core.pixelbased.componentgeocoding.interpolator" and as value geodetic and euclidian. This allows more interpolators in the future.

Reply:
done